### PR TITLE
Add ACS test

### PR DESCRIPTION
**Pull request by bitboom**
This PR
- adds [ACS; Architecture Compliance Suite](https://github.com/ARM-software/cca-rmm-acs) for RMM as a third-party project
- add a test script ~~and attaches it to CI~~

## Note
We use ACS's version 0.7 and it needs some patches to test easily.
Please refer [cca-rmm-acs branch](https://github.com/Samsung/islet/tree/3rd-cca-rmm-acs-230531)

## How to run ACS test

```
$ scripts/tests/acs.sh 
[!] Building realm management monitor...: tf-rmm
[!] Building ACS...
[!] Building bootloaders(bl1.bin, fip.bin)... 
REGRESSION REPORT: ................
==================
   TOTAL TESTS     : 59
   TOTAL PASSED    : 41
   TOTAL FAILED    : 16
   TOTAL SKIPPED   : 2
   TOTAL SIM ERROR : 0

******* END OF ACS *******
```

or you can test directly like

```sh
$ scripts/fvp-cca --normal-world=acs --rmm=tf-rmm
```

### CODE REVIEW

File: scripts/config.py [hunk 1]
```diff
@@ -56,4 +60,5 @@
 LINUX_CROSS_COMPILE = os.path.join(ROOT, "assets/toolchain/aarch64-none-linux-gnu/bin/aarch64-none-linux-gnu-")
 KVMTOOL_CROSS_COMPILE = os.path.join(ROOT, "assets/toolchain/aarch64-none-linux-gnu-10-2/bin/aarch64-none-linux-gnu-")
 FASTMODEL = os.path.join(ROOT, "assets/fastmodel/Base_RevC_AEMvA_pkg/models/Linux64_GCC-9.3")
+FVP_BIN = os.path.join(FASTMODEL, "./FVP_Base_RevC-2xAEMvA")
```
Comment (isaac2-lee) [inline scripts/config.py:63]: if you made it, can you apply it on script?

Comment (bitboom) [reply scripts/config.py:63]: It was applied. Could you please check [here](https://github.com/Samsung/islet/pull/81/files#diff-b1b563f761715532e6c757b98e58b7300afff98e8c4047a78154d92f41b36fa0R300)?

Comment (isaac2-lee) [reply scripts/config.py:63]: I missed it. Thanks for your kind

**Review by isaac2-lee — APPROVED** [review]
approved this pull request.

**Review by jinbpark — APPROVED** [review]

**Review by jinbpark — APPROVED** [review]

**Review by jinbpark — APPROVED** [review]

**Pull request merged**